### ISSUE TITLE
xref crash

### ISSUE THREAD

**Issue by ivanstepanovftw** [radareorg/cutter#1483]
**Environment information**
* Operating System: 5.1.0-1-MANJARO
* Package manager: pacman
* Cutter version: 1.8.0.r37.g67f865af-1
* radare2 version: 3.4.1 0 @ linux-x86-64 git.3.4.1
* QT5 version: 5.12.2-1
* File format: ELF 64-bit LSB executable, x86-64, version 1 (GNU/Linux), statically linked, for GNU/Linux 3.2.0

**Describe the bug**
Crash

**To Reproduce**

1. Open file
1. Ok
1. Strings tab
1. Select any string and Ctrl+X
1. No any xref found, close xref window
1. It crashed 
[crash1.log](https://github.com/radareorg/cutter/files/3101936/crash1.log)

Okay, lets try again:
1. Open file
1. Ok
1. Strings tab
1. Select any string and Ctrl+X
1. Follow by xref by double-click
1. It crashed
[crash2.log](https://github.com/radareorg/cutter/files/3101937/crash2.log)

**Expected behavior**
Follow xref

**Screenshots**

**Additional context**

**Comment by ivanstepanovftw** [comment]
Logs updated, sorry for inconvenience

**Comment by ivanstepanovftw** [comment]
Execution was: 
```
gdb -batch -ex "run" -ex "thread apply all bt" `which Cutter` >~/Desktop/crash1.log 2>&1
```

**Comment by ITAYC0HEN** [comment]
can you please check with the most recent version of Cutter, from git.
You report mentioned it is Cutter 1.8.0 while we are already in 1.8.1 and fixed many bugs :)

**Comment by ivanstepanovftw** [comment]
Yes, fixed in Cutter 1.8.1. Why 1.8.0 with so many bugs used in manjaro/pacman? Who is responsible for this?

**Comment by Maijin** [comment]
Greetings @ivanstepanovftw,

We are not responsible for the distribution you're using the details of the maintainer can be found here https://www.archlinux.org/packages/community/x86_64/radare2-cutter/. 

Best Regards,
